tests/integration/test_charm.py

```diff
@@ -107,27 +108,23 @@ async def test_setup_discourse(
 
     # Get the form info
     assert parsed_registration.body
-    utf8 = parsed_registration.body.find("input", attrs={"name": "utf8"}).get("value")  # type: ignore
-    authenticity_token = parsed_registration.body.find("input", attrs={"name": "authenticity_token"}).get("value")  # type: ignore
-    form_fields = {
-        "utf8": utf8,
-        "authenticity_token": authenticity_token,
-        "username": "admin",
-        "email": app_config["developer_emails"],
-        "password": "MyLovelySecurePassword2022!",
-    }
-
-    form_headers = {
-        "Host": f"{app_config['external_hostname']}",
-        "Content-Type": "application/x-www-form-urlencoded",
-    }
-
     logger.info("Submitting registration form")
 
     response = session.post(
         f"{discourse_url}/finish-installation/register",
-        headers=form_headers,
-        data=urlencode(form_fields),
+        headers={
+            "Host": f"{app_config['external_hostname']}",
+            "Content-Type": "application/x-www-form-urlencoded",
+        },
+        data=urlencode(
+            {
+                "utf8": parsed_registration.body.find("input", attrs={"name": "utf8"}).get("value"),  # type: ignore # pylint: disable=line-too-long
```

Comment:
For type: ignore, should we add a comment on why? This is requested by our contributing guide: https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#static-code-analysis

Reply:
This also applies to any other type: ignore that were added

Reply:
I think we should address that, however I haven't authored the ignores, they were part of other PRs, perhaps on a subsequent refactor?

Reply:
I'll leave that to you as a judgement call - it might be good to do it sooner rather than later to clean up the code base

Reply:
I'm seeing such ignores piling up in other repos as well, let's have a discussion on the topic, we should address the root of why `mypy` is complaining and how to fix it. I'm dropping it until we figure out a holistic approach for all of our charm repos.